style.css

```diff
@@ -0,0 +1,206 @@
+*{
+    margin:0;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+.grid{
+    display: grid;
+    grid-template: "left main right";
+    grid-template-columns: 1fr 800px 1fr ;
+    
+}
+
+.header{
+    display:grid;
+    padding-top: 15px;
+    font-weight: 500;
+    grid-template-rows: repeat(auto-fill);
+}
+.header > hr{
+    width:20px;
+}
+
+
+/* right bar */
+
+.rightsidebar{
+    grid-area: right;
+    display: grid;
+    grid-template-rows: 1fr;
+    grid-template-columns: 1fr;
+    border: solid 1px #e6b0b0ab;
+    
+}
+.tweet-feed{
+    display: grid;
+    grid-template-columns: 800px;
+    grid-template-rows: 1fr;
+    
+}
+.rightsidebar > .Rcontainer{
+    display: grid;
+    grid-template-rows: 75px 1fr 1fr;
+   
+}
+.searchbar{
+    background-color: rgba(168, 168, 172, 0.226);
+    max-width: 200px;
+    border-radius: 50px;
```

Comment:
These 3 properties are also in your .newsfeed, .follow   maybe reusing classes could be applied here too?